README.md

```diff
@@ -44,8 +44,6 @@ that includes it. Once it reaches the end of its lifespan, the experiment will b
 | [`InMemoryChatMessageStore`][7]                                                                                                                                                                                                                                                              | Memory Store                             | December 2024     | None         | <a href="https://colab.research.google.com/github/deepset-ai/haystack-cookbook/blob/main/notebooks/conversational_rag_using_memory.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/> | [Discuss](https://github.com/deepset-ai/haystack-experimental/discussions/75)  |
 | [`Auto-Merging Retriever`][8] & [`HierarchicalDocumentSplitter`][9]                                                                                                                                                                                                                          | Document Splitting & Retrieval Technique | December 2024     | None         | <a href="https://colab.research.google.com/github/deepset-ai/haystack-cookbook/blob/main/notebooks/auto_merging_retriever.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>          | [Discuss](https://github.com/deepset-ai/haystack-experimental/discussions/78)  |
 | [`LLMMetadataExtractor`][13]                                                                                                                                                                                                                                                                 | Metadata extraction with LLM             | December 2024     | None         | <a href="https://colab.research.google.com/github/deepset-ai/haystack-experimental/blob/main/examples/metadata_extraction_with_llm_metadata_extractor.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>                                   | [Discuss](https://github.com/deepset-ai/haystack-experimental/discussions/130) |
-| [`AsyncPipeline`][17]                                                                                                                                                                                                                                                                 | Async Pipeline & Components | February 2024     | None         | <a href="https://colab.research.google.com/github/deepset-ai/haystack-experimental/blob/main/examples/async_pipeline.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/>               | [Discuss](https://github.com/deepset-ai/haystack-experimental/discussions/152) |
```

Comment:
@mathislucka Can you add ready-made super components and `Agent` to the list?

Reply:
I will do that in separate PRs. I just want to get this one merged so that I don't get merge conflicts (see https://github.com/deepset-ai/haystack-experimental/issues/211)